### PR TITLE
User metadata code

### PR DESCRIPTION
**Pull request by chregu**

### CODE REVIEW

File: src/Image.php [hunk 1]
```diff
@@ -342,6 +344,53 @@ public function deleteDynamicMetadata($dynamicMetadataName, $hash, $organization
...
+            self::SOURCEIMAGE_RESOURCE,
+            $this->getOrganization($organization),
+            $hash,
+            self::USER_META_RESOURCE
+        ]);
+        $data = [];
+        if ($fields) {
+            foreach ($fields as $key => $value) {
+                if ($value instanceof \DateTime) {
+                    $fields[$key] = $value ->setTimezone(new \DateTimeZone("UTC"))->format("Y-m-d\TH:i:s.v\Z");
```
Comment (thePanz) [inline src/Image.php]: There's an extra space before ` ->setTimezone`, can you fix that?

Comment (chregu) [reply src/Image.php]: fixed

**Comment by dbu** [comment]
is this a BC break? did static metadata get removed?

**Comment by thePanz** [comment]
The `static_metadata` was never set and there were no API to set/get it.

**Comment by dbu** [comment]
ah ok, good then. sorry, it just looked a bit weird in the diff

**Pull request merged**